core-plugin/testSrc/com/google/gct/idea/appengine/validation/MethodParameterTypeInspectionTest.java

```diff
@@ -24,6 +24,14 @@
  */
 public class MethodParameterTypeInspectionTest extends EndpointTestBase {
 
+  @Override
```

Comment:
so the fake JDK is not available in tests? Do you mean the fake JDK is not in the binary intellij distribution so it's missing when configured as the IDEA SK?